client/host.go

```diff
@@ -72,9 +72,9 @@ func FindHostForTests(hostId string, host *Host) {
 	*host = queriedHost
 }
 
-func (c *Client) GetHostsByPoolName(pool string, tags ...interface{}) (hosts []map[string]interface{}, err error) {
+func (c *Client) GetHostsByPoolName(host Host) (hosts []map[string]interface{}, err error) {
```

Comment:
Since the new version of the function no longer uses a pool name, I think we should update its name to reflect its current functionality. How about `GetHosts(host Host)`?